CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/display/GraphVisualAccess.java

```diff
@@ -42,6 +43,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import org.openide.util.NbPreferences;
```

Comment:
Is this an unused import too?

Reply:
See latest commit "Resolve PR changes" Yes, It is.